saleor/wombat/__init__.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+import requests
+import logging
+
+logger = logging.getLogger('wombat')
+
+
+class WombatClient(object):
+    endpoint = 'https://push.wombat.co'
```

Comment:
Endpoint should be in settings as well with fallback to this value. I am not familiar with the `wombat` but similar services have different endpoints for testing.